core/templates/dev/head/pages/exploration_editor/editor_tab/StateSolution.js

```diff
@@ -61,7 +61,10 @@ oppia.controller('StateSolution', [
 
     $scope.getSolutionSummary = function() {
       var solution = stateSolutionService.savedMemento;
-      return solution.getSummary(stateInteractionIdService.savedMemento);
+      var solutionAsPlainText =
+        solution.getSummary(stateInteractionIdService.savedMemento);
+      solutionAsPlainText = solutionAsPlainText.replace(/&quot;/g,'');
```

Comment:
Add a space after the comma.

Also, can this use the convertToPlainText filter instead? Better to not duplicate if possible.

Reply:
I purposely didn't use the filter. I thought it will be a dependency and might unnecessarily affect the performance, although I'm not sure about it.

Reply:
Ah, OK. No, I think it's OK, if you use $filter('convertToPlainText')(...). Where it becomes a performance problem is when it's used in the HTML directly.